ConsoleApp1/Weather.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConsoleApp1
+{
+    internal class Weather
+    {
+        [JsonProperty("location")]
+        private Results location;
+
+        [JsonProperty("current")]
+        private Current current;
+     
+        private void PrintWeather()
+        {
+            location.LocationInfo();
+            current.CurrentWeatherInfo();
+        }
+
+        private void GetWeatherForecast(string city)
+        {
+            HttpClient client = new HttpClient();
```

Comment:
HttpClient closures system resource so to prevent dataleak use `using HttpClient client = new HttpClient();`